R/gh_gql.R

```diff
@@ -13,6 +13,20 @@
 #' @seealso [gh()] for the GitHub v3 API.
 #' @examplesIf FALSE
 #' gh_gql("query { viewer { login }}")
+#' Get rate limit
+#' ratelimit_query <- "query {
+#'   viewer {
+#'     login
+#'   }
+#'   rateLimit {
+#'     limit
+#'     cost
+#'     remaining
+#'     resetAt
+#'   }
+#' }"
+#' 
+#' gh::gh_gql(ratelimit_query)
```

Comment:
Thanks! Can you please lose the `gh::` prefix, for consistency? Thanks.

Reply:
Absolutely. Done.